CMakeLists.txt

```diff
@@ -86,9 +86,6 @@ if(NOT MSVC)
     set_directory_properties(PROPERTIES INCLUDE_DIRECTORIES ${DL_INCLUDE_DIR})
     add_definitions(-DHAVE_DL=1)
   endif()
-else() #MSVC
-  # This flag enables multi process compiling for Visual Studio 2005 and above
-  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
```

Comment:
On 2 Oct 2015 18:03, "Sergei Nikulov" <notifications@github.com> wrote:
> @mloskot If you think it's important, will return it back. Anyway cmake
do almost job for build and I just trying reduce number of non portable
scripts.
>

No need to restore it.
In fact, cutting down on number of CMake script lines is a very good idea
(including any custom macros or logic scripted specifically for SOCI).